pages.es/linux/yum.md

```diff
@@ -0,0 +1,28 @@
+# yum
+
+> Administrador de paquetes para RHEL, CentOS y Fedora (para versiones anteriores).
+> Más información: <https://man7.org/linux/man-pages/man8/yum.8.html>.
```

Comment:
We use this source now, could you please update the English and Persian pages too?
```suggestion
> Más información: <https://manned.org/yum>.
```